File: backend.api/Domain/Entities/Book.cs
﻿namespace Domain.Entities;

public class Book
{
    public int? Id { get; set; }
    public int? UserId { get; set; }
    public string? Title { get; set; }
    public string? Author { get; set; }
    public bool? IsCompleted { get; set; }
}

File: backend.api/models/book.go
package models

import "go.mongodb.org/mongo-driver/bson/primitive"

type Book struct {
	Id     primitive.ObjectID `json:"_id,omitempty" bson:"_id,omitempty"`
	UserId primitive.ObjectID `json:"userid"`
	Title  string             `json:"title"`
}
